lib/generators/rspec/controller/templates/request_spec.rb

```diff
@@ -1,7 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe "<%= class_name.pluralize %>", <%= type_metatag(:request) %> do
-<% namespaced_path = regular_class_path.join('/') %>
+<% namespaced_path = regular_class_path.join('/') -%>
+<% if actions.empty? -%>
+  describe "GET /index" do
+    it "returns http success" do
```

Comment:
This is a failing example rather than a pending one.
There's a difference in how `pending` works inside an example and when used to declare an example.
When used inside an example, it expects the example to fail, and marks it as pending. If the example doesn't fail, it ... fails.

So I suggest changing this to:
```ruby
  describe "GET /index" do
    pending "add some scenarios (or delete) #{__FILE__}"
  end
```